website/docs/r/maintenance_window.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: maintenance_window"
+sidebar_current: "docs-mongodbatlas-resource-maintenance_window"
+description: |-
+    Provides an Maintenance Window resource.
+---
+
+# mongodbatlas_maintenance_window
+
+`mongodbatlas_maintenance_window` provides a resource to take a scheduled maintenance event for a project up to two times. 
+Deferred maintenance events occur during your preferred maintenance window exactly one week after the previously scheduled date and time.
+
```

Comment:
Add from our core docs the following: 

Maintenance Window Considerations: 
* Urgent Maintenance Activities Cannot Wait: Urgent maintenance activities such as security patches cannot wait for your chosen window. Atlas will start those maintenance activities when needed.

Once maintenance is scheduled for your cluster, you cannot change your maintenance window until the current maintenance efforts have completed.
* Maintenance Requires Replica Set Elections: Atlas performs maintenance the same way as the manual maintenance procedure. This requires at least one replica set election during the maintenance window per replica set.
* Maintenance Starts As Close to the Hour As Possible: Maintenance always begins as close to the scheduled hour as possible, but in-progress cluster updates or expected system issues could delay the start time.